src/styles/_catalog.scss

```diff
@@ -19,4 +19,20 @@
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, $card-size);
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-size);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-size);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-size);
+  }
```

Comment:
you do not need 4 media here, set one `grid-template` by default

Reply:
Fixed